File: Jukebox/Jukebox/Playlist.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.Data.Entity.Spatial

Partial Public Class Playlist
    Public Sub New()
        Songs = New HashSet(Of Song)()
    End Sub

    Public Property ID As Integer

    Public Property Creator As String

    Public Overridable Property Songs As ICollection(Of Song)
End Class

File: Jukebox/Jukebox/Jukebox/Models/Playlist.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Data.Entity.Spatial;


namespace Jukebox {

    public partial class Playlists
    {
        public int ID { get; set; }
        public string Name { get; set; }
        public string Creator { get; set; }
        public string SongList { get; set; }
        public int DurationMinutes { get; set; }
        public int DurationSeconds { get; set; }
    }
}